src/commands/upstack-commands/onto.ts

```diff
@@ -14,7 +14,7 @@ const args = {
 
 export const command = "onto <branch>";
 export const description =
-  "Rebase any upstack branches onto the latest commit (HEAD) of the current branch.";
+  "Rebase all upstack branches onto the latest commit (HEAD) of the target branch.";
```

Comment:
Technically HEAD is reserved for where you currently are in the repo (afaik)

```suggestion
  "Rebase all upstack branches onto the latest commit (tip) of the target branch.";
```